src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokemon.java

```diff
@@ -11,7 +12,7 @@
     private String pokemonNumber;
     private String pokemonName;
     private String pokemonImg;
-    private List<String> pokemonType;
+    private List<PokemonType> pokemonType;
```

Comment:
you can use an enum to store all the possible types of pokemon. see dto/enums package I added.

Reply:
Very good.  Using String as a type is often an indication of object oriented design not implemented properly.  Change to an enum from a String, make this more OOD.